Tests/JSONAPITests.swift

```diff
@@ -317,6 +317,13 @@ class JSONAPISpec: QuickSpec {
                 expect(included).to(beNil())
             }
             
+            it("should not include if an array of entities don't have relationships") {
```

Comment:
not 100% sure that this is what we need, we had to check the Element type exactly for this reason (include also when array was empty but not nil)...